### PR TITLE
Fix for 2230. Document BackingField Attribute.

### PR DESCRIPTION
**Pull request by lajones**

### CODE REVIEW

File: entity-framework/core/modeling/backing-field.md [hunk 1]
```diff
@@ -26,6 +26,12 @@ Note that backing fields are only discovered for properties that are included in
 
 You can also configure backing fields explicitly, e.g. if the field name doesn't correspond to the above conventions:
```
Comment (smitpatel) [inline entity-framework/core/modeling/backing-field.md:27]: Should also change this line.

Comment (lajones) [reply entity-framework/core/modeling/backing-field.md]: Done

**Comment by lajones** [comment]
Fixes #2230

**Comment by smitpatel** [comment]
Please use "Squash and merge" option in UI to merge pull request to master.

**Pull request merged**